### ISSUE TITLE
Why explicitly set booleans to false?

### ISSUE THREAD

**Issue by ZLightning** [substack/minimist#94]
undefined is already falsy.  By setting the boolean to false, you lose the ability to tell if false was explicitly set or just the default value when the argument was not specified.

The "default" object can only set default values for string arguments, so you can not force the default to undefined.

Is this project still being maintained?

**Comment by YounGoat** [comment]
I agree with @ZLightning that there is tiny difference between DEFAULT and ABSENT. In package [commandos](https://www.npmjs.com/package/commandos), I use decorator DEFAULT if indicate that a default value is really need when and ONLY when the option is absent.

```javascript
// FILENAME: test.js

const commandos = require('commandos');
const options = commandos.parse({ options: [ '--name', '--age DEFAULT unknown', '--version DEFAULT 1' ] });
// Only options which are multiple, or with default values, 
// or set in CLI will occur in the returned value.

// Suppose that the command is: node ./test.js --age
options.version === '1'; 
options.age === true; // NOT "unknown"
typeof options.name == 'undefined'; // NOT false
```

**Comment by kling90** [comment]
I was wondering the same, expected to get `undefined` when no default value is specified in options.
